Update how to guide of protocol buffers (#292)
ballerina-platform/ballerina-dev-website: v1-2/learn/how-to-generate-code-for-protocol-buffers.md

}
```

> * You can run the following command to generate the service template file.

Reviewer: ```suggestion
* Execute the below command to generate the service template file.
```

```
$ ballerina grpc --input helloworld_service.proto --mode service --output service
```

> Once you execute the command, the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.

Reviewer: ```suggestion
Once you execute the command, the service template file (`helloWorld_sample_service.bal`) gets generated inside the service directory.
```

Reviewer: Can change as "Once you execute the command, the service template file (`helloWorld_sample_service.bal`) is generated inside the service directory."

> Note: If you have multiple services in Protocol Buffers definition, this command will generate a stub file with common message types and service template file for each service definition. This is to avoid duplicating message types in all service files.

Reviewer: ```suggestion
> **Note:** If you have multiple services in a Protocol Buffers definition, this command will generate a stub file with common message types and a service template file for each service definition. This is to avoid duplicating message types in all the service files.
```

> * You can run the following command to generate the client/service stub and client template.

Reviewer: ```suggestion
* Execute the below command to generate the client/service stub and client template.
```

```
$ ballerina grpc --input helloworld_service.proto --mode client --output client
```
Once you execute the command, the stub file(`helloworld_service_pb.bal`) and the service template file (`helloWorld_sample_client.bal`) are generated inside the client directory.

> * You can run the following command to generate only the client/service stub.

Reviewer: ```suggestion
* Execute the below command to generate only the client/service stub.
```

}
```

* Execute the below command to generate the service template file.
```
$ ballerina grpc --input helloworld_service.proto --mode service --output service
```

> Once you execute the command, the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.

Author: ```suggestion
Once you execute the command, the service template file (`helloWorld_sample_service.bal`) is generated inside the service directory.
```
